metadata/v1.24.0/proposal_1_upgrade_framework.json

```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Multi-step proposal to upgrade mainnet framework, version v1.24.0",
+  "description": "This includes changes in https://github.com/aptos-labs/aptos-core/releases/tag/aptos-node-v1.24.0-rc",
+  "source_code_url": "https://github.com/aptos-foundation/mainnet-proposals/tree/main/sources/v1.24.0/proposal_1_upgrade_framework",
```

Comment:
Note: I manually set this URL based on the PR comment here: https://github.com/aptos-foundation/mainnet-proposals/pull/123#discussion_r1848494855